gammapy/estimators/points/tests/test_sed.py

```diff
@@ -390,6 +391,21 @@ def test_run_map_pwl_reoptimize(fpe_map_pwl_reoptimize):
     assert_allclose(actual, [9.788123, 0.486066, 17.603708], rtol=1e-2)
 
 
+@requires_dependency("iminuit")
+@requires_data()
+def test_run_template_npred(fpe_map_pwl, tmpdir):
+    datasets, fpe = fpe_map_pwl
+    dataset = datasets[0]
+    models = Models(dataset.models)
+    model = TemplateNPredModel(dataset.background, datasets_names=[dataset.name])
+    models.append(model)
+    dataset.models = models
+
+    fpe.run(dataset)
+    fpe.sum_over_energy_groups = True
+    fpe.run(dataset)
+
```

Comment:
No numerical test is needed here? We test just that it runs and not that it runs well?

Reply:
added